patdel0/dev-digest
# Create Result card

Waiting on https://github.com/patdel0/dev-digest/pull/63; Relates #16 

This PR creates the MVP version of the component for the search results. It includes the rating functionality, updating the database whenever the value is changed.

Out of this piece of work the following tasks were created:
https://github.com/patdel0/dev-digest/issues/66; https://github.com/patdel0/dev-digest/issues/65

<img width="640" alt="image" src="https://github.com/patdel0/dev-digest/assets/53922624/9f75569d-ef7b-4f86-98c4-269fbb36ae18">

## How to Test
1. `./script/server`
2. `./script/test`
3. Confirm the [Story](http://localhost:6006/?path=/story/components-resultcard--default) is working as expected
4. Search for a term in the [Homepage](http://localhost:3000/)
5. Upvote or downvote an article, notice the new rating
6. Refresh the page
7. Repeat step 4 for the same term
8. Confirm the rating is the same as step 5
